README.md

```diff
@@ -63,9 +63,85 @@ Observer pattern. (2022, November 13). Wikipedia. https://en.wikipedia.org/wiki/
 #### References
 
 
+## Functional Patterns vs. OOP Design Patterns in JavaScript
 
+### Introduction: 
 
+In traditional Object-Oriented Programming (OOP), design patterns are widely used to solve common problems by providing structured approaches to design and development. However, with the rise of Functional Programming (FP), there's a shift towards different paradigms and patterns. In this Section, we'll explore the contrast between OOP design patterns and functional patterns in JavaScript.
 
+### OOP Design Patterns
+OOP design patterns like Singleton, Factory, Observer, and others are well-established concepts in JavaScript development. They focus on class hierarchies, encapsulation, and inheritance to achieve modularity and extensibility.
+
+For example, the Singleton pattern ensures that a class has only one instance and provides a global point of access to it. In JavaScript:
+

```

Comment:
I think that rather than pasting code here without really commenting on it, it would be better if you just provided a link to the implementation folder. That way, we don't have to worry about this example here going stale. And the important part of your contribution is really in the next section anyway.

Please remove this JS code and just add a hyperlink in line 75 pointing to the JS Singleton folder.